src/SelectMulti.vue

```diff
@@ -83,14 +83,14 @@
 				default: () => null
 			},
 			/**
-             * Determines whether or not typing in the input will filter the provided options.
+			 * Determines whether or not typing in the input will filter the provided options.
 			 * Should be set to false if you are making API calls to fetch options.
```

Comment:
@rbvea should it be set to false if you're making api calls to fetch options while typing? So, if I make an initial call to get options, but don't make any subsequent calls based on the user input, I should set this to true?

Reply:
also, have you noticed a huge drop in performance when this is set to true?

Reply:
@tcrite1 yes, when you load options after the initial render, this is the setting to set to `true`. I haven't noticed a drop in performance, have you?

Reply:
@rbvea when this is set to true, I have noticed a drop in performance. It's not related to this change though. Also, our list is like 800 options :-/

Reply:
I'm seeing that y'all are using it for the [SearchFilters](https://github.com/politico/delta/blob/develop/src/client/src/components/search/SearchFilters.vue). Is it a "dumb" implementation — in that it's derived from the facets returned from Sail? If so then I don't think the flag is needed. 

Over in Pro we use it whenever :options can vary. In politico/search, there's a concept of "[currentOptions](https://github.com/politico/legos-vue3/blob/0a831317e308267ad9e872043d29f79117d93148/src/components/PoliticoSearch/components/GeneratedSelectMulti.vue#L76-L84)" pared with "clientEscapeHatches", which [fetches the dropdown options from autocomplete](https://github.com/politico/reggie/blob/16b25b819f466e359d7374d693632b003d308f56/src/client/src/components/search/filters/clientEscapeHatchesMapping.ts#L27-L30).


Only if :options are populated by autocomplete, then the prop should be set to true.

Reply:
It's working okay for the search filters. it's this one where I'm seeing the issue?

https://github.com/politico/delta/blob/develop/src/client/src/components/shared/UserShare.vue